### PR TITLE
Module 1 Exercise 1 Pull request

### PR DESCRIPTION
**Pull request by MikeLaBou**
Catching up on things

### CODE REVIEW

**Comment by Jonesey13** [comment]
Merging in for demonstration purposes.

File: appbkp.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from flask import Flask, render_template, request, redirect, url_for
```
Comment (hugh-emerson) [inline appbkp.py:1]: This file should be deleted, right?

File: excersize_1_pull_test [hunk 2]
```diff
@@ -0,0 +1 @@
+Starting here OOOOOooo
```
Comment (hugh-emerson) [inline excersize_1_pull_test:1]: This file should also be deleted, right?

File: session_items.py [hunk 3]
```diff
@@ -67,3 +61,7 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def remove_item(id):
+    session['items'] = [item for item in get_items() if item['id'] == int(id)]
```
Comment (hugh-emerson) [inline session_items.py:66]: This currently removes all items except the specified one

File: app.py [hunk 4]
```diff
@@ -6,7 +6,26 @@
...
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', 
+        items=sorted(session.get_items(), key=lambda item: item['status'], reverse=True))
+
+@app.route('/create', methods=['POST'])
+def create():
+    session.add_item(request.form['title'])
+    return index()
```
Comment (hugh-emerson) [inline app.py:15]: You should actually `return redirect(url_for('index'))`

Same for the two functions below.

The difference is that you send a redirect response to the user's browser, which will then make a fresh request to get the home page again. Main benefits are so the user sees the correct URL and refreshing the page will not resubmit the form.

**Review by hugh-emerson — COMMENTED** [review]
Generally looking good. I've left some small suggestions, but the easiest and most impactful is to keep the codebase tidy by not merging your unused, experimental files

**Pull request merged**